File: src/tp01/ej01/Gear.java
package tp01.ej01;

public class Gear {
    private final int chainring;
    private final double cog;

    public Gear(int chainring, double cog) {
        this.chainring = chainring;
        this.cog = cog;
    }

   public double getRatio(){
        return this.chainring / this.cog;
   }
}

File: src/tp08/ej01/gear.rb
require_relative "wheel"

class Gear 
  def initialize(chainring, cog, rim = 0, tire = 0)
    @chainring = chainring
    @cog = cog
    @wheel = Wheel.new(rim, tire)
  end
  
  def ratio
    @chainring / @cog.to_f
  end

  def gear_inches
    ratio * @wheel.diameter
  end
end
